etc/WMAgentConfig.py

```diff
@@ -50,9 +50,13 @@
 # List of BossAir plugins that this agent will use.
 bossAirPlugins = ["CondorPlugin"]
 
-## Plugin with Condor-Python API ...
+# Plugin with Condor-Python API ...
 bossAirPlugins = ["PyCondorPlugin", "CondorPlugin"]
```

Comment:
I am sorry Alan, I actually meant remove this line 54. I guess you removed it first time and I made you put it back. (I thought you added that) Sorry about the mistake.